rest/route/version_create.go

```diff
@@ -19,11 +19,11 @@ func makeVersionCreateHandler(sc data.Connector) gimlet.RouteHandler {
 }
 
 type versionCreateHandler struct {
-	ProjectID string          `json:"project_id"`
-	Message   string          `json:"message"`
-	Active    bool            `json:"activate"`
-	IsAdHoc   bool            `json:"is_adhoc"`
-	Config    json.RawMessage `json:"config"`
+	ProjectName string          `json:"project_name"`
```

Comment:
While I don't think anyone is using this currently, changing the request format like this breaks backwards compatibility, so we shouldn't do this unless there's a good reason

Reply:
good point